### DISCUSSION TITLE
[docs] Update README with instructions for use in EAS Build

### DISCUSSION THREAD

**Discussion by gregfenton** [henninghall/react-native-date-picker#647] [General]
It would be great if the README gave instructions on how to use this cool library with EAS Build + Managed Workflow.  As it is currently documented, it seems to lead newcomers towards using the Bare Workflow and/or the Expo "classic build".

**Comment by henninghall** [comment]
Hi, I don't have much experience with expo myself so it would be very appreciated if you could send a PR with readme improvements that clarify this!
